app/src/main/res/menu/options.xml

```diff
@@ -31,6 +31,11 @@
     <item
         android:id="@+id/daynight"
         android:title="@string/switch_theme" />
+
+    <item
+        android:id="@+id/helpMe"
+        android:title="Help" />
```

Comment:
hardcoded string; also maybe calling it "Tutorial" or "Take a tour" would be better?